src/styles/card.css

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+  padding-top: 32px;
+}
+
+.card__image {
+  display: flex;
+  flex-grow: 1;
+
+  margin: 0 var(--card-image-horizontal-margins) 40px;
```

Comment:
please add paddings to card element instead of adding margins for each elements
Should be smth like it
![image](https://user-images.githubusercontent.com/54936044/224472097-b1fef94d-182c-40b4-b8ae-823372ac7847.png)